man/man1/p11sak.1.in

```diff
@@ -1544,6 +1544,83 @@ option to export the certificate(s) in binary (DER-encoded) form. Default is
 PEM format.
 .PP
 .
+.SS "Extracting the public key of certificates"
+.
+.B p11sak
+.BR extract\-cert\-pubkey | extr\-pubkey | extpub
+.RI [ CERTTYPE ]
+.BR \-\-slot | \-s
+.I SLOTID
+.RB [ \-\-pin | \-p
+.IR PIN ]
+.RB [ \-\-force\-pin\-prompt ]
+.RB [ \-\-label | \-L
+.IR LABEL ]
+.RB [ \-\-id | \-i
+.IR ID ]
+.RB [ \-\-attr | \-a
+.IR ATTRS ]
+.RB [ \-\-force | \-f ]
```

Comment:
The following option are missing here:

 -A, --new-attr ATTRS
 -l, --new-label LABEL  
 -I, --new-id ID    

You describe them below in the text, but don't list them here.

Reply:
Done.